RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# Release
+
+The kubernetes-base-addons repository is intended to support a minimal base set of Addons required to provide monitoring, logging, alerting, and backups for every version of Kubernetes actively supported by the upstream Kubernetes community.
+It is not intended to be tied to any specific version of Kubernetes installer (ie. Konvoy or Kommander), or Kubeaddons.
+Changes to Konvoy or Kommander to create a resource needed by an Addon should be avoided.
+
+- [Schedule](#schedule)
+- [Considerations](#considerations)
+  - [Kubernetes](#kubernetes)
+  - [Branches and Tags](#branches-and-tags)
+- [Process](#process)
+  - [Testing Release (Weekly, Thursday)](#testing-release-weekly-thursday)
+  - [Stable Release (Weekly, Wednesday)](#stable-release-weekly-wednesday)
+
+## Schedule
+
+Releases should be weekly on Wednesdays, or as needed to address CVEs.
+
+## Considerations
+
+### Kubernetes
+
+The Addons in this repo are the minimum base set of supported<sup>[1](#footnote1)</sup> Addons to be installed as a suite for any supported version of Kubernetes.
+
+Kubernetes support must be maintained for the latest general release of Kubernetes and the two prior minor releases.
+At the time of this writing, the latest Kubernetes release is 1.17.2.
+Support from this repo, at this time, should cover 1.15.x, 1.16.x, and 1.17.x.
+
+### Branches and Tags
+
+As much as possible, we will try to maintain a `master` branch that is compatible with all supported versions of Kubernetes.
+If there is a variation in the Kubernetes API which requires a _**breaking change**_ to the Addon, a _**branch**_ will be made for the prior Kubernetes versions.
```

Comment:
Are there any restrictions to what can go in build tag bump, vs a patch and minor releases?

Reply:
At this time, the expectation is that we're going to be extremely strict about the upgrade processes, requiring it to be transparent to the user. The usual semver reasons for major/minor/patch versions will not apply to this release process.

It wouldn't really make sense, anyway, since it's the underlying addons that are changing semver releases and we can't really match pace with those.

For now, I'm leaving this out of this document. We can revisit if the need arises.